packages/@o3r/schematics/src/interfaces/angular-workspace.ts

```diff
@@ -29,44 +29,45 @@ export interface WorkspaceLayout {
 
 
 export interface WorkspaceSchematics extends SchematicOptions {
-  /** @deprecated */
+  /** @deprecated will be removed in v10 */
   '@otter/ng-tools:api-service'?: {
     path: string;
   };
-  /** @deprecated */
+  /** @deprecated will be removed in v10 */
   '@otter/ng-tools:component'?: {
     path: string;
   };
-  /** @deprecated */
+  /** @deprecated will be removed in v10 */
   '@otter/ng-tools:service'?: {
     path: string;
   };
-  /** @deprecated */
+  /** @deprecated will be removed in v10 */
   '@otter/ng-tools:store'?: {
     path: string;
   };
-  /** @deprecated */
+  /** @deprecated will be removed in v10*/
```

Comment:
isn't it too soon to deprecate these? Shouldn't we wait for v12?

Reply:
Not for schematics, let's deprecate the schematics in the next marjor.
Otherise it will be a nightmare